src/hotspot/cpu/aarch64/c1_Defs_aarch64.hpp

```diff
@@ -87,4 +87,9 @@ enum {
   pd_arraycopystub_reserved_argument_area_size = 2
 };
 
+// multiplier factor to update reserved_area_argument_size
+enum {
+  pd_reserved_argument_area_size_factor = 2
+};
```

Comment:
[Not a review, just a drive-by comment.]
Please don't use enums to define constants like this.  This is an old and obsolete style that shouldn't be
used any more.  Quoting from the HotSpot Style Guide:
"Due to bugs in certain (very old) compilers, there is widespread use of enums and avoidance of in-class initialization of static integral constant members. Compilers having such bugs are no longer supported. Except where an enum is semantically appropriate, new code should use integral constants."

Reply:
@kimbarrett Where to see `HotSpot Style Guide`, can you give a URL link?

Reply:
@sunny868 here: https://github.com/openjdk/jdk/blob/master/doc/hotspot-style.md

Reply:
@kimbarrett I've made changes, please take a look now. 

Thank you.